README.md

```diff
@@ -96,6 +96,7 @@ yarn dev
 - Pro Tip:
   - Explore scripts within `package.json` for more build options
   - Get **faster** production builds by building only one language. E.g. in your `.env` file, set `BUILD_LOCALES=en` to build the content only in English
+  - To build the site in other languages too, you need to set them in `BUILD_LOCALES`, eg: `` BUILD_LOCALES=`en,es` `` if you also want to include Spanish
```

Comment:
```suggestion
  - To build the site in other selected languages too, you need to set them in `BUILD_LOCALES`, eg: `BUILD_LOCALES=en,es` if you also want to build only English (required) and Spanish.
  - To build all languages, simply comment this line out with a hash mark, eg: `# BUILD_LOCALES=`
```

Reply:
@wackerow `BUILD_LOCALES=en,es` is not the same, backticks needs to be around the locales

Reply:
Oh interesting... so personally I have never used the back-ticks around this list of locales (and it still works just fine). These are not required as everything after the `=` will be treated as a string anyway. For simplicity here I'd recommend we just leave them off.